app/_locales/en/messages.json

```diff
@@ -1,14 +1,14 @@
 {
   "appName": {
-    "message": "Behave!",
+    "message": "Behave! — Offensive Comments Blocker",
```

Comment:
@iislucas Think the goal for this change is to make the app more accessible to the wider audience. "Controler" sounds like a complex thing right away. "Filter" maybe?